BuildingCodeParser.nb

```diff
@@ -0,0 +1,2129 @@
+(* Content-type: application/vnd.wolfram.mathematica *)
```

Comment:
I don't agree with using StartProcess and ProcessObjects. There are a lot of platform-specific edge cases and there are problems with their implementations.

I would rather documentation for building, installing, and testing be in MarkDown .md files.

Reply:
That's fine, I can keep this notebook for myself ... I like having a single notebook which builds, installs, and tests everything. Feel free to reject the pull request.